client/src/components/DataFiles/DataFilesModals/DataFilesAddProjectModal.module.scss

```diff
@@ -0,0 +1,8 @@
+.root {
+  display: inherit;
+}
```

Comment:
__Ineffectual Style__

This style is ineffectual.

- If it is meant to solve something, then please check that it is solved.
- If it is cruft, then please check that it is solved.

```suggestion
```

![Display Style Ineffectual](https://user-images.githubusercontent.com/62723358/105412231-0f34c400-5bfa-11eb-8ac6-47ea8b17ed4f.png)